integration_tests/run_pyspark_from_build.sh

```diff
@@ -46,27 +46,48 @@ else
 
     # support alternate local jars NOT building from the source code
     if [ -d "$LOCAL_JAR_PATH" ]; then
-        AVRO_JARS=$(echo "$LOCAL_JAR_PATH"/spark-avro*.jar)
-        PLUGIN_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark_*.jar)
-        if [ -f $(echo $LOCAL_JAR_PATH/parquet-hadoop*.jar) ]; then
+        if [ -f "$LOCAL_JAR_PATH"/spark-avro*.jar ]; then
```

Comment:
on the one hand it's good to check for file existence, but on the other hand we don't take any action if it does not exist. There is no error message. And the ALL_JARS assignment on L106 does not  seem to care about it either. Do we need this change?